### DISCUSSION TITLE
Question about LEADER_KEY_STRICT_KEY_PROCESSING

### DISCUSSION THREAD

**Discussion by bogorad** [manna-harbour/miryoku#306] [Q&A] [answered]
I'm not sure this is the right place to ask this, but no one has responded in [/r/olkb](https://www.reddit.com/r/olkb/comments/1bj1g0o/leader_key_strict_key_processing_or_something_else/) so far, and I'm using Miryoku, so here goes:

I want to catch `TAB` while any of the right-hand HRMs are active, and do some housekeeping. I have `TAB` mapped to `LT(U_MOUSE,KC_TAB)`.

I added TAB to `process_record_user`, but it never arrives. Instead, I get keycode for `LT(U_MOUSE,KC_TAB)` - (I debug-print it).

```
LT(U_MOUSE,KC_TAB) 0x452b
KL: kc: 0x452B, col:  5, row:  3, pressed: 0, time: 11855, int: 0, count: 1
```

I checked the QMK docs and found `LEADER_KEY_STRICT_KEY_PROCESSING` which causes this behavior, but it isn't defined (I debug-printed via #ifdef).

What else might be causing this behavior? Thanks!

**Comment by manna-harbour** [comment, marked as answer]
The best place to ask non-Miryoku QMK questions is the Help channel on the QMK discord. You can find the link from https://github.com/manna-harbour/miryoku/discussions/8. When asking a non-Miryoku QMK question on the QMK discord it's best to not mention Miryoku or they'll just send you back here... You can try making your change to the default keymap for your keyboard so you have a non-Miryoku example of the issue you can link to. You could also try searching for `process_record_user` and `LT` etc. in the QMK discord, or searching the QMK codebase (at the point before user keymaps were removed).

I suspect getting the `LT` is expected and you need to examine something else (`record->tap.count`) to determine if it's the tap or the hold.
